src/OpenLoco/src/Ui/WindowManager.h

```diff
@@ -259,6 +259,11 @@ namespace OpenLoco::Ui::Windows
         void close(Ui::Window* window);
     }
 
+    namespace ObjectLoadError
+    {
+        Window* open(std::vector<ObjectHeader> list);
```

Comment:
Do you mean to copy the list? Probably want to const ref

Reply:
Ah, I didn't realise this would be copying. Nice catch, will amend.